test/integration/targets/ansible-galaxy/aliases

```diff
@@ -0,0 +1 @@
+posix/ci/group3
```

Comment:
Please include a trailing newline. It makes the diff cleaner in the future if another line is appended.

Reply:
@mattclay one of my vim plugins strips trailing whitespace, is this going to be a common request and I should alter my vim config or is this limited in scope to the `aliases`?

Reply:
It's a general recommendation, not limited to `aliases` files.

Reply:
@mattclay I pushed a new commit with an empty line at the end of the file, but for some reason it's not showing in the diff on GitHub ... but when I open my local copy of the file it has an empty line at the bottom of the file and `git diff` says I'm good. I'm not sure what to do here.

Reply:
I'm seeing two newlines now, but there should only be one.